inspire/modules/workflows/tasks/submission.py

```diff
@@ -86,11 +86,17 @@ def _create_ticket(obj, eng):
         rt_queue = cfg.get("CFG_BIBCATALOG_QUEUES") or queue
         sip = d.get_latest_sip(sealed=False)
         subject = u"Your suggestion to INSPIRE: {0}".format(d.title)
+        try:
+            user_comment = filter(lambda k: 'submitter' == k.get('source', ''),
+                                  sip.metadata.get("hidden_note", []))[0].get('extra_comments', None)
```

Comment:
will this work? According to https://github.com/inspirehep/inspire-next/pull/192/files#diff-3b59804856efabb4ecf3b800ac348ee9R533 there is no key ``extra_comments`` inside of ``hidden_note`` ?